### PR TITLE
Add html-proof for automated testing

### LINKED ISSUE #124: 'For delegates' broken
'For delegates' button at top right pf page takes you to page not found.

### LINKED ISSUE #152: Programme link in footer broken
The link to programme in the footer currently leads to an index listing of the programme folder. 
At the moment the programme link in the main menu leads to the *call for contributions* so a quick fix could include also linking it there.

### LINKED ISSUE #153: eMail addresses have no mailto address
eMail addresses are currently rendered nicely but the mailto link does not contain the actual email address.

### PR DESCRIPTION
**Pull request by Chilipp**
This should have been done before, but I forgot about it. 

As we are facing a lot of broken links, this PR adds a checker using [html-proofer](https://github.com/gjtorikian/html-proofer/) that particularly checks for broken links. Note that it unfortunately does not work for setting the `baseurl` or `url` in the `_config.yml`, so I needed to exclude it in the `.circleci/_config.yml` file.

closes #124, closes #152, closes #153 and solves several other broken links

### CODE REVIEW

**Comment by vsoch** [comment]
And some background is here https://github.com/USRSE/usrse.github.io/issues/171

**Comment by Chilipp** [comment]
Nice @vsoch! I did not know this. Having you as the developer of the testing library onboard is already reason enough for me to use your library due to the better support :wink: would you mind changing the files in this PR to set it up for url-checker? Or shall we do this in a new PR?

**Comment by vsoch** [comment]
If it's okay with you, I think a new PR should be done - this one has quite a bit going on (unless it wasn't rebased?)

**Comment by Chilipp** [comment]
alright, then let's close this, I make a cherry-picked one for the fixes (https://github.com/RSE-leaders/SORSE20/pull/161/commits/788ad47c21e810bfc7951e9113ef7f6e5a1d7370, https://github.com/RSE-leaders/SORSE20/pull/161/commits/21bca57be1ad7cc7f0a3606d591975d7fdfd659d, https://github.com/RSE-leaders/SORSE20/pull/161/commits/97b73871005d16147cb1d851e34af07f93bf998f, https://github.com/RSE-leaders/SORSE20/pull/161/commits/082bf523603f1ce4ac84affed91e93d8a42b9a4b and https://github.com/RSE-leaders/SORSE20/pull/161/commits/f31d7f18dc7a5aeae79fcb59136325fbb6811655) and you make a new one for implementing the URL checker. Alright?

**Comment by vsoch** [comment]
If you are going to close, let's just leave it open so I can make sure they work the same. Also, I'm going to bed soon so I hope you don't expect this soon! Feel free to jump in and do the PR if you don't want to wait for me :)

**Comment by Chilipp** [comment]
Sure, it must be really late across the ocean, no worries :D I'd at least already implement the fixes, the url checker can wait until you're awake again, I guess (as the broken links where identified already). Thanks a lot @vsoch!

**Comment by vsoch** [comment]
@Chilipp this should be un-draft-ified!

**Comment by Chilipp** [comment]
alright. I merged the master branch and fixed the new problems. So this is ready to be reviewed

File: _pages/privacy.md [hunk 1]
```diff
@@ -36,7 +38,7 @@ toc_sticky: true
 ## Event registration
 
 - We use a Google form to submit to the topic bazaar, but this is also stated
-  separately [here](https://rse-leaders.github.io/SORSE20/programme/bazaar/#submit).
+  separately [here]({% include fix-link.html link=submit_topic %}).
```
Comment (vsoch) [inline _pages/privacy.md:41]: Just to check - I saw this twice in the repo - did we get both of them?

Comment (Chilipp) [reply _pages/privacy.md:41]: I only fixed one of them for which I got the error. Where is the other?

Comment (Chilipp) [reply _pages/privacy.md:41]: looking for `rse-leaders` within all repository files did not show anything

**Comment by vsoch** [comment]
My mistake! I did a grep and two showed up (one inside, one outside the _site folder). What about the 404 page url, that was the other url that was missing:
```
orry, but the page you were trying to view does not exist --- perhaps you can try searching for it below.
Sorry, but the page you were trying to view does not exist.

<script>
  var GOOG_FIXURL_LANG = 'en';
  var GOOG_FIXURL_SITE = '{{ site.url }}'
</script>
<script src="https://linkhelp.clients.google.com/tbproxy/lh/wm/fixurl.js">
</script>
```
I'm not sure https://linkhelp.clients.google.com/tbproxy/lh/wm/fixurl.js is online anymore.

**Comment by vsoch** [comment]
Tests pass, so all must be good :)

**Comment by Chilipp** [comment]
true, I excluded 404.html in the check. I'll add your 404.html from https://github.com/SORSE/sorse.github.io/pull/183/commits/e7682cfca24895712e3be5b72632f5ad8aeb014a

**Comment by Chilipp** [comment]
done, thanks @vsoch!

**Comment by Chilipp** [comment]
feel free to approve and merge this PR then @vsoch

File: README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ As we cannot host one version of this site with github pages (generated from the
 3. `cd` to the clone of this repository
 4. Run `bundle install`
 5. Serve the site locally via `bundle exec jekyll serve`
-6. Visit the site in the browser via http://localhost:4000
+6. Visit the site in the browser via http://localhost:4000/SORSE20
```
Comment (vsoch) [inline README.md:34]: Wouldn't this just be the base url now, since we are hosting at sorse.github.io?

Comment (Chilipp) [reply README.md]: Done with 61ec875, thanks for spotting this.

File: README.md [hunk 3]
```diff
@@ -1,30 +1,49 @@
 # SORSE Website repository
 
-This repository contains the source code for the Series of Online Research Software Events 2020. It is based on the [Minimal Mistakes Jekyll theme](https://github.com/mmistakes/minimal-mistakes) and hosted with Github Pages. The static website is build automatically on Github but you can also build it locally.
+This repository contains the source code for the Series of Online Research Software
+Events 2020. It is based on the [Minimal Mistakes Jekyll theme](https://github.com/mmistakes/minimal-mistakes)
+and hosted with Github Pages. The static website is build automatically on Github
+but you can also build it locally.
 
-The documentation how to contribute is in the [Wiki](wiki) of this repository. There you find, how to add new members, new national chapters, how to assign someone to a specific team, etc.
+The documentation how to contribute is in the [Wiki](wiki) of this repository.
```
Comment (vsoch) [inline README.md:8]: Will this wiki link work?

File: README.md [hunk 4]
```diff
@@ -1,30 +1,49 @@
...
+New to Github? Don't worry, you'll figure it out fast and it's no problem if it
+does not immediately work. If you have a problem, ask in slack or create an issue.
+You should have a look into the [Projects](https://github.com/SORSE/sorse.github.io/projects)
+tab (see above), that gives you an overview on the current tasks.
 
 ## Important facts
 
-- if you have are lacking a link and nevertheless want to add the text as a placeholder, mark it with `{: .missing}`. The link will then be highlighted in red (example: `[some important document](){: .missing}`)
-- when referencing internal link, e.g. `{{ site.data.committee.national_chapters.deRSE.internal }}`, use prepend it with a `{{ site.baseurl }}`, otherwise it will not resolve correctly on github. In otherwords
+- if you have are lacking a link and nevertheless want to add the text as a
```
Comment (vsoch) [inline README.md:31]: Have or lacking a link?

Comment (Chilipp) [reply README.md:31]:
```suggestion
- if you are lacking a link and nevertheless want to add the text as a
```

File: README.md (hunk 3 above)
Comment (Chilipp) [reply README.md:8]:
```suggestion
The documentation how to contribute is in the [Wiki](https://github.com/SORSE/sorse.github.io/wiki) of this repository.
```

**Comment by Chilipp** [comment]
done @vsoch, these issues existed already in the original README :sweat_smile:

File: README.md [hunk 5]
```diff
@@ -1,30 +1,49 @@
...
 ## How to contribute
 Unless you know exactly what you are doing, please always
 
-- do not push directly to the master branch (usually you are anyway not allowed to do so). You must contribute via pull request
+- do not push directly to the master branch (usually you are anyway not allowed
+  to do so). You must contribute via pull request
 - we always require a review from another person (four-eye-principle)
 - assign issues and pull request to applicable milestones, labels and projects.
-- if your working on an issue or PR, assign it to you! If you are not working on it anymore, remove your assignment from the PR, either because you are finished or because you don't have the time to do more, unassign yourself and ideally leave a message that someone else should take over.
+- if your working on an issue or PR, assign it to you! If you are not working on
```
Comment (vsoch) [inline README.md:19]: Sorry one more here, “you’re” instead of “your”

**Comment by vsoch** [comment]
And I should probably stop looking, now I just feel bad :) I think it’s good for squash and merge and any remaining details can be addressed later! Perfection is the enemy of the good, something something.

File: README.md (hunk 5 above)
Comment (Chilipp) [reply README.md:19]:
```suggestion
- if you're working on an issue or PR, assign it to you! If you are not working on
```

**Pull request merged**